src/main/java/meditracker/medication/MedicationManager.java

```diff
@@ -49,12 +50,28 @@ protected static void clearMedication() {
      * Adds a Medication to the list of Medication
      *
      * @param medication Medication to be added to the list
+     * @throws MediTrackerException When a duplicate medication is found
      */
-    public static void addMedication(Medication medication) {
+    public static void addMedication(Medication medication) throws MediTrackerException{
```

Comment:
Please add a space before the opening `{`

Reply:
Resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/218/commits/6bad8f6bb17510a2145b6373685aa1dfcef58c3a